src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -245,17 +251,12 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
           </ClaimTable>
 
           <AccountClaimSummary>
+            <AccountDetails isClaimer label="Claiming with account" account={account} connectedAccount={account} />
```

Comment:
I thought this a few times, would it be more clear to say `Investing account` or `Paying account` to highlight who is the one incurring to costs and which one benefits

Reply:
Two things could be happening (at the same time or not):
- You are **claiming** an airdrop.
- You are **[buying||investing in]** vCOW tokens.

Also the connected account is the account **paying** for both of the above.

So not sure what word we should prioritize here, in order to be as consistent as possible. I can image something like **'Executing account'** covering all and being more agnostic. But feel free to change wording here.

Reply:
What about one word shorter:

```suggestion
            <AccountDetails isClaimer label="Claiming with" account={account} connectedAccount={account} />
```